unit_tests/test_simulation_config.py

```diff
@@ -44,19 +44,23 @@ def test_assertion_is_raised_if_incorrect_info_is_provided(self):
             get_simulation_config()
 
     def test_default_simulation_info_is_under_setup_module_if_base_path_isnt_provided(self):
-        set_simulation_file_env(base_setup_path=None, simulation_config_path='test.json',
-                                run_on_redis=False)
+        os.environ["SIMULATION_CONFIG_FILE_PATH"] = \
```

Comment:
I think this way to break lines is somehow encouraged by some PyCharm default settings. However, it is actually discouraged by PEP8:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
In this case, this could become:
```python
os.environ["SIMULATION_CONFIG_FILE_PATH"] = create_simulation_config_path(
            base_setup_path=None, simulation_config_path='test.json', run_on_redis=False)
```

Reply:
https://github.com/gridsingularity/d3a-api-client/pull/94/commits/502901535a99d23ce71dc26480ac8f9c7da5a49e